src/utils/minify.js

```diff
@@ -5,6 +5,8 @@ export function minify(code) {
       .trim()
       // Remove comments
       .replaceAll(/<!--.*-->/g, "")
+      // Replace border-*-width:0 or none with border-*:0;
+      .replaceAll(/border-(top|right|bottom|left|\*)-width:\s*(0|none);/g, "border-$1:0;")
```

Comment:
question: what's `|\*` stand for? Maybe a misunderstanding?

```suggestion
      .replaceAll(/border-(top|right|bottom|left)-width:\s*(0|none);/g, "border-$1:0;")
```